pkgs/development/python-modules/tesserocr/default.nix

```diff
@@ -0,0 +1,35 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+  # build dependencies
+  cython,
+  leptonica,
+  pkg-config,
+  tesseract,
+  # extra python packages
+  pillow
+}:
+
+buildPythonPackage rec {
+  pname = "tesserocr";
+  version = "2.5.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0nj33mwvppacy6p5mqk9a4x26hx9ailshgad84ks60wyms6rgjiv";
+  };
+
+  buildInputs = [ cython leptonica tesseract pkg-config ];
```

Comment:
```suggestion
  nativeBuildInputs = [ cython leptonica tesseract pkg-config ];
```

Reply:
Putting `tesseract` and `leptonica` in `nativeBuildInputs` makes the build fail.

Reply:
sounds good, i was only sure about pkg-config and cython :)